src/Applications/LDAS_App/LDASsa_DEFAULT_inputs_ensupd.nml

```diff
@@ -2064,6 +2064,228 @@ obs_param_nml(48)%xcorr          = 0.1875
 obs_param_nml(48)%ycorr          = 0.1875
 obs_param_nml(48)%adapt          = 0
 
+! --------------------------------------------------------------------
+!
+!  49 = ASCAT_META_SM_A (ASCAT soil moisture ascending)
+!
+! https://navigator.eumetsat.int/product/EO:EUM:DAT:METOP:SOMO25
+
+obs_param_nml(49)%descr          = 'ASCAT_META_SM_A'
+obs_param_nml(49)%orbit          = 1
+obs_param_nml(49)%pol            = 0
+obs_param_nml(49)%N_ang          = 0
+obs_param_nml(49)%freq           = 0
+obs_param_nml(49)%FOV            = 20.
+obs_param_nml(49)%FOV_units      = 'km'
+obs_param_nml(49)%assim          = .false.
+obs_param_nml(49)%scale          = .false.
+obs_param_nml(49)%getinnov       = .false.
+obs_param_nml(49)%RTM_ID         = 0
+obs_param_nml(49)%bias_Npar      = 0
+obs_param_nml(49)%bias_trel      = 864000
+obs_param_nml(49)%bias_tcut      = 432000
+obs_param_nml(49)%nodata         = -9999.
+obs_param_nml(49)%varname        = 'sfds'
+obs_param_nml(49)%units          = '%'
+obs_param_nml(49)%path           = '/discover/nobackup/projects/gmao/merra/iau/merra_land/DATA/ASCAT_EUMETSAT/Metop_A/'
+obs_param_nml(49)%name           = 'M02-ASCA-ASCSMO02'
+obs_param_nml(49)%scalepath      = ''
+obs_param_nml(49)%scalename      = ''
+obs_param_nml(49)%flistpath      = ''
+obs_param_nml(49)%flistname      = ''
+obs_param_nml(49)%errstd         = .04
```

Comment:
@amfox37 : I think we should change this entry to be consistent with the units (%) listed above.  Fundamentally, the numbers here should all be consistent, so if the sfds data are in percent, then we should also have the errstd in %. I suppose it would be something like 9.0 (approx 0.04 m3/m3 / porosity * 100, with porosity = 0.45).  

The scaling routine for the ASCAT obs converts the obs value from % to m3/m3, and it should also convert the "errstd" number from % to m3/m3.  We may be able to derive a location-specific conversion factor from the info provided by the ASCAT scaling params.  Note that for AMSR-E sm retrievals, we also convert the input errstd (by multiplying with the ratio of the time series std-devs, as in:
`errstd_scaled = err_std * std_model / std_obs`
where std_model and std_obs are the time series std-devs of the model and obs, resp.  See subroutine scale_obs_sfmc_cdf(). This isn't perfect, but it is a rough way of accounting for the difference in the dynamic range of the model and obs (and, in the case of ASCAT, also for the difference in units).  I hope this makes sense.

Reply:
Apologies, I missed that the errstd is already rescaled in `scale_obs_sfmc_zscore()`.  The errstd entry in the nml file still needs changing, though.

Reply:
add /100. after initial read of [errstd](https://github.com/GEOS-ESM/GEOSldas/blob/3f95ee02780cf30259b9a2ea85eceb000822a7e7/src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90#L1954) during tile super-obbing
```fortran
             ASCAT_sm_std(ii) = this_obs_param%errstd/100.    ! change units from percent (0-100) to fraction (0-1)
```